### PR TITLE
fix(dashboard,medusa,fulfillment): Move Shipping Profiles to settings

### PR DESCRIPTION
**Pull request by kasperkristensen**
**What**
- Moves Shipping Profiles to settings
- Adds `q` and filters to shipping profile list endpoint
- Adds new details page for profiles

### CODE REVIEW

File: packages/admin-next/dashboard/src/v2-routes/shipping-profiles/shipping-profile-detail/shipping-profile-detail.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+export const ShippingProfileDetail = () => {
+  const { id } = useParams()
+
+  const { shipping_profile, isLoading, isError, error } = useShippingProfile(
+    id!
+  )
+
+  if (isLoading || !shipping_profile) {
+    return <div>Loading...</div>
```
Comment (sradevski) [inline packages/admin-next/dashboard/src/v2-routes/shipping-profiles/shipping-profile-detail/shipping-profile-detail.tsx]: I assume we'll tackle the loaders at a different point?

File: packages/admin-next/dashboard/src/v2-routes/shipping-profiles/shipping-profiles-list/components/shipping-profile-list-table/shipping-profile-list-table.tsx [hunk 2]
```diff
@@ -1,29 +1,36 @@
...
   })
 
   const { shipping_profiles, count, isLoading, isError, error } =
-    useShippingProfiles({
-      ...searchParams,
-    })
+    useShippingProfiles(
+      {
+        ...searchParams,
+      },
```
Comment (sradevski) [inline packages/admin-next/dashboard/src/v2-routes/shipping-profiles/shipping-profiles-list/components/shipping-profile-list-table/shipping-profile-list-table.tsx]:
```suggestion
     searchParams,
```

**Review by sradevski — APPROVED** [review]
Looks good overall!

**Review by fPolic — APPROVED** [review]
LGTM

**Pull request merged**